raiden/network/transport/matrix/transport.py

```diff
@@ -903,7 +915,8 @@ def _has_multiple_partner_addresses(self, room: Room) -> bool:
             address = validate_userid_signature(user)
             if address != self._raiden_service.address:
                 joined_addresses.add(address)
-                if len(joined_addresses) > 1:
+                # if address is None it means a false displayname, should leave
```

Comment:
because of this, a more descriptive method name would be `_has_invalid_address_in_room`

Reply:
a room with 2 valid addresses should also return true